aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -323,6 +380,26 @@ module aptos_framework::fungible_asset {
         store_exists(object::object_address(&store)) && borrow_store_resource(&store).frozen
     }
 
+    #[view]
+    /// Return whether a fungible asset type is dispatchable.
+    public fun is_dispatchable<T: key>(store: Object<T>): bool acquires FungibleStore {
+        let fa_store = borrow_store_resource(&store);
+        let metadata_addr = object::object_address(&fa_store.metadata);
+        exists<DispatchFunctionStore>(metadata_addr)
+    }
+
+    public(friend) fun deposit_function<T: key>(store: Object<T>): FunctionInfo acquires FungibleStore, DispatchFunctionStore {
```

Comment:
I would rename this to deposit_dispatch_function, so it cannot be confused with the rest (same as withdraw)

long name is fine, as it is just internal function